### PR TITLE
fix: LEAP-508: Add `noreferrer` to external links

### PR DESCRIPTION
**Pull request by hlomzik**
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#security_and_privacy), `noopener` is not required anymore, because it's implied automatically
> setting target="_blank" implicitly provides the same protection as setting rel="noopener"

However `rel="noreferrer"` is important to not reveal anything for 3rd parties. And there is a security issue because of this: https://github.com/HumanSignal/label-studio/security/code-scanning/769

This PR adds `noreferrer` to external links like slack, github or pypi. It doesn't add it to links to our documentation.
Also it adds `target="_blank"` to external links that didn't have it.

Changed links:
- Menu bar — github, slack, pypi
- Export modal — github, slack
- Invite people — docs
- Webhooks — docs
- Errors about wrong resource in LSF — link to this resource

#### Does this change affect security?
Fixes https://github.com/HumanSignal/label-studio/security/code-scanning/769

#### What alternative approaches were there?
[Add `noopener`](https://github.com/HumanSignal/label-studio/pull/5609) but that doesn't make it fully secure

### CODE REVIEW

**Comment by hlomzik** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8633444704)
> Successfully merged:  233 files changed, 14814 insertions(+), 11986 deletions(-)

**Review by nick-skriabin — APPROVED** [review]

File: web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx [hunk 1]
```diff
@@ -235,7 +235,13 @@ const FormatInfo = ({ availableFormats, selected, onClick }) => {
       <Elem name="feedback">
         Can't find an export format?
         <br/>
-        Please let us know in <a className="no-go" href="https://slack.labelstud.io/?source=product-export">Slack</a> or submit an issue to the <a className="no-go" href="https://github.com/heartexlabs/label-studio-converter/issues">Repository</a>
+        Please let us know in
+        {" "}
+        <a className="no-go" href="https://slack.labelstud.io/?source=product-export" target="_blank">Slack</a>
+        {" "}
+        or submit an issue to the
+        {" "}
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx:243]: this looks like we're using spaces to adjust positioning/spacing - im guessing the styles applied to the `<a>` is not showing the right amount of space (probably no spacing at all)
this should be addressed in the the style sheet

Comment (hlomzik) [reply web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx:243]: it's not about adding paddings, this way we add usual spaces trimmed by react

Comment (yyassi-heartex) [reply web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx:243]: looking into this, its not so much that spaces around tags are removed as much as it is removing spaces at the beginning and end of the line - I dont particularly like this approach but having this paragraph on one line isnt much better - I can see personal preference coming into play here so i'll go ahead and resolve the convo/approve the pr

**Review by yyassi-heartex — APPROVED** [review]

File: web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx (hunk 1 above)
Comment (hlomzik) [reply web/apps/labelstudio/src/pages/ExportPage/ExportPage.jsx:243]: sorry to correct you, but leading/trailing spaces are stripped only around tags, if you have usual multilined text, lines will be separated by spaces in final text. but not if line starts or ends with tag.

**Pull request merged**